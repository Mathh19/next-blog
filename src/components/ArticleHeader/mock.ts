import { ArticleHeaderProps } from '.';

export default {
  title:
    'Sint deserunt officia sit ipsum. Minim commodo sit ea labore sit sit ex ad tempor esse labore.',
  excerpt:
    'Proident sunt est enim fugiat sit incididunt sunt veniam adipisicing amet aliquip.',
  cover:
    'https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg',
  createdAt: '2022-09-02T15:16:09.246+00:00',
  author: {
    id: '123456789',
    attributes: {
      slug: 'matheus-freitas',
      name: 'Matheus Freitas',
    },
  },
  categories: [
    {
      id: 'asdslgyh65',
      attributes: {
        name: 'React',
        slug: 'react',
      },
    },
    {
      id: 'asdslgyh66',
      attributes: {
        name: 'JavaScript',
        slug: 'javascript',
      },
    },
  ],
} as ArticleHeaderProps;
