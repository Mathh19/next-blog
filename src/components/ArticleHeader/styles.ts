import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.header`
  ${({ theme }) => css`
    padding-bottom: ${theme.spacings.xlarge};
    margin-bottom: ${theme.spacings.xlarge};
    border-bottom: 0.1rem solid ${theme.colors.mediumGray};

    ${HeadingStyles} {
      margin: 0;
      margin-bottom: ${theme.spacings.medium};
    }
  `}
`;

export const Excerpt = styled.p`
  ${({ theme }) => css`
    margin: ${theme.spacings.medium} 0;
    font-size: ${theme.font.sizes.medium};
  `}
`;

export const CoverContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: center;
    margin-bottom: ${theme.spacings.medium};
  `}
`;

export const Cover = styled.img`
  ${css`
    max-width: 100%;
    display: block;
  `}
`;
