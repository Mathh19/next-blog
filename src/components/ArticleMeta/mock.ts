import { ArticleMetaProps } from '.';

export default {
  createdAt: '2022-09-02T15:16:09.246+00:00',
  author: {
    id: '1234567890',
    attributes: {
      slug: 'matheus-freitas',
      name: 'Matheus Freitas',
    },
  },
  categories: [
    {
      id: 'asdslgyh65',
      attributes: {
        name: 'React',
        slug: 'react',
      },
    },
    {
      id: 'asdslgyh66',
      attributes: {
        name: 'JavaScript',
        slug: 'javascript',
      },
    },
  ],
} as ArticleMetaProps;
