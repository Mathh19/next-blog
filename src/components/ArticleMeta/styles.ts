import styled, { css } from 'styled-components';

export const Wrapper = styled.article`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.small};
    color: ${theme.colors.darkGray};
    font-style: italic;

    .categories span::after {
      content: ', ';
    }
    .categories span:last-child::after {
      content: '';
    }

    a {
      color: ${theme.colors.secondary};
      text-decoration: none;
      transition: all 300ms ease-in-out;

      &:hover {
        filter: brightness(50%);
      }
    }
  `}
`;
