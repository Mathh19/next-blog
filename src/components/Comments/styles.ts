import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    max-width: ${theme.sizes.content};
    width: 100%;
    margin: ${theme.spacings.large} auto;
    padding: 0 ${theme.spacings.large};
  `}
`;

export const NoComments = styled.div`
  ${({ theme }) => css`
    text-align: center;
    color: ${theme.colors.secondary};
    font-size: ${theme.font.sizes.medium};
  `}
`;
