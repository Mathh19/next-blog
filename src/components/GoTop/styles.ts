import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    background-color: #082032;
    color: ${theme.colors.secondary};
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
    opacity: 0.7;
    border-radius: 50%;
    border: 0.2rem solid ${theme.colors.secondary};
    &:hover {
      transition: 500ms ease-in-out all;
      background-color: #12324a;
      opacity: unset;
    }
  `}
`;
