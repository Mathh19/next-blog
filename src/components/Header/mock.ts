import { HeaderProps } from '.';

export default {
  srcImg: 'assets/images/cat-test.png',
  title: 'Titulo blog',
  description:
    'Ea aliqua labore anim minim voluptate est in cillum qui consequat cupidatat adipisicing ex in. Amet velit consequat mollit deserunt consectetur dolor exercitation commodo consequat voluptate eu non commodo ipsum.',
  showDescription: true,
} as HeaderProps;
