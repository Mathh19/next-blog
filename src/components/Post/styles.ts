import styled, { css } from 'styled-components';
export const Wrapper = styled.div`
  ${({ theme }) => css`
   img {
    border: 1px solid ${theme.colors.secondary};
  }
  `}
`;
