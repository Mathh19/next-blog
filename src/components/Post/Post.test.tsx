import { renderTheme } from '../../styles/render-theme';
import { Post, PostProps } from '.';
import mock from './mock';
import { screen } from '@testing-library/react';

const props: PostProps = mock;

describe('<Post />', () => {
  it('should render header, excerpt, cover, metadata and post', () => {
    const { container } = renderTheme(<Post {...props} />);

    expect(
      screen.getByRole('heading', { name: props.data[0].attributes.title }),
    ).toBeInTheDocument();
    expect(
      screen.getByRole('img', { name: props.data[0].attributes.title }),
    ).toBeInTheDocument();
    expect(
      screen.getByText(
        /^Proident sunt est enim fugiat sit incididunt sunt veniam adipisicing amet aliquip/i,
      ),
    ).toBeInTheDocument();
    expect(
      screen.getAllByText(
        /^Proident sunt est enim fugiat sit incididunt sunt veniam adipisicing amet aliquip/i,
      )[0],
    ).toHaveStyle({ 'font-size': '2.4rem' });
    expect(container).toMatchSnapshot();
  });
});
