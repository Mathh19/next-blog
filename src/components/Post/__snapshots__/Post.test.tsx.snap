// Vitest Snapshot v1

exports[`<Post /> > should render header, excerpt, cover, metadata and post 1`] = `
.c0 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 96rem;
}

.c8 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 80rem;
}

.c3 {
  color: #082032;
  line-height: 1.2;
  font-size: 4.0rem;
  text-transform: none;
}

.c7 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c7 .categories span::after {
  content: ', ';
}

.c7 .categories span:last-child::after {
  content: '';
}

.c7 a {
  color: #DC143C;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c7 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c1 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c1 .c2 {
  margin: 0;
  margin-bottom: 2.4rem;
}

.c4 {
  margin: 2.4rem 0;
  font-size: 2.4rem;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-bottom: 2.4rem;
}

.c6 {
  max-width: 100%;
  display: block;
}

.c9 {
  font-size: 1.8rem;
  line-height: 1.5;
}

.c9 p {
  margin: 2.4rem 0;
}

.c9 a {
  color: #DC143C;
}

.c9 a:visited,
.c9 a:link {
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c9 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c9 pre {
  background-color: #2C394B;
  padding: 2.4rem;
  font-family: monospace;
  color: #B2C8DF;
  margin: 4.0rem 0;
  width: 100%;
  font-size: 1.6rem;
  overflow-x: auto;
}

.c9 pre code {
  background: inherit;
  color: inherit;
}

.c9 code {
  font-family: monospace;
  color: #B2C8DF;
  font-size: 1.6rem;
  background: #2C394B;
  padding: 0.2rem;
  margin: 0.2rem;
  border-radius: 5px;
}

.c9 img {
  max-width: 100%;
}

.c9 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c9 .image figcaption {
  font-size: 1.6rem;
  color: #000000;
  padding: 1.6rem;
  text-align: center;
  line-height: 1.3;
}

.c9 .image-style-side {
  float: right;
  max-width: 50%;
  margin: 2.4rem;
}

.c9 hr {
  border: none;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c9 ul,
.c9 ol {
  margin: 2.4rem;
}

.c9 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c9 table {
  width: 100%;
  border-collapse: collapse;
  margin: 2.4rem 0;
}

.c9 table td,
.c9 table th {
  padding: 1.6rem;
  border: 0.1rem solid #DDDDDD;
}

.c9 blockquote {
  border-left: 0.5rem solid #DC143C;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

@media (max-width:768px) {
  .c9 {
    font-size: 2rem;
  }

  .c9 .image-style-side {
    float: none;
    max-width: 100%;
    margin: 0;
  }
}

<div>
  <div
    class=""
  >
    <div
      class="c0"
    >
      <header
        class="c1"
      >
        <h1
          class="c2 c3"
        >
          Sint deserunt officia sit ipsum. Minim commodo sit ea labore sit sit ex ad tempor esse labore.
        </h1>
        <p
          class="c4"
        >
          Proident sunt est enim fugiat sit incididunt sunt veniam adipisicing amet aliquip.
        </p>
        <div
          class="c5"
        >
          <img
            alt="Sint deserunt officia sit ipsum. Minim commodo sit ea labore sit sit ex ad tempor esse labore."
            class="c6"
            src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
          />
        </div>
        <article
          class="c7"
        >
          <p>
            <span>
              Por 
            </span>
            <a
              href="/author/matheus-freitas"
            >
              Matheus Freitas
            </a>
            <span
              class="separator"
            >
               | 
            </span>
            <time
              datetime="2022-09-02T15:16:09.246+00:00"
            >
              2 de set. de 2022
            </time>
            <span
              class="separator"
            >
               | 
            </span>
            <span
              class="categories"
            >
              <span>
                <a
                  href="/category/react"
                >
                  React
                </a>
              </span>
              <span>
                <a
                  href="/category/javascript"
                >
                  JavaScript
                </a>
              </span>
            </span>
          </p>
        </article>
      </header>
    </div>
    <div
      class="c8"
    >
      <div
        class="c9"
      >
        <pre>
          <code>
            &lt;p&gt;Nostro &lt;strong&gt;definitiones&lt;/strong&gt; (&lt;i&gt;React&lt;/i&gt;) sed - &lt;u&gt;Matheus Freitas&lt;/u&gt;. Sea &lt;code&gt;useEffect&lt;/code&gt;, useState dicat fabulas &lt;a href="https://github.com/Mathh19"&gt;consetetur&lt;/a&gt; no, odio deseruisse assueverit mei eu, vis probo nonumes molestie id. Ipsum movet liberavisse an mea, diceret vituperata ne his, delenit apeirian quo ea. Imperdiet percipitur te eos. Per ludus aliquam eu. &lt;a href="https://github.com/Mathh19"&gt;Eos ex tincidunt necessitatibus, timeam theophrastus mel &lt;/a&gt;. La.&lt;/p&gt;
&lt;h1&gt;Tincidunt necessitatibus, timeam theophrastus mel &lt;/h1&gt;
&lt;h2&gt;Tincidunt necessitatibus, timeam theophrastus mel &lt;/h2&gt;
&lt;h3&gt;Tincidunt necessitatibus, timeam theophrastus mel &lt;/h3&gt;
&lt;h4&gt;Tincidunt necessitatibus, timeam theophrastus mel &lt;/h4&gt;
&lt;h5&gt;Tincidunt necessitatibus, timeam theophrastus mel &lt;/h5&gt;
&lt;h6&gt;Tincidunt necessitatibus, timeam theophrastus mel &lt;/h6&gt;
&lt;pre&gt;&lt;code class="language-plaintext"&gt;import { HtmlContent } from '../HtmlContent';
import * as Styled from './styles';
export type FooterProps = {
  footerHtml: string;
};
export const Footer = ({ footerHtml }: FooterProps) =&gt; {
  return (
    &lt;Styled.Container&gt;
      &lt;HtmlContent html={footerHtml} /&gt;
    &lt;/Styled.Container&gt;
  );
};
&lt;/code&gt;&lt;/pre&gt;
&lt;p&gt;Qui eu &lt;i&gt;mutat&lt;/i&gt; exerci &lt;span style="background-color:hsl(60,75%,60%);"&gt;dolores&lt;/span&gt;, his no &lt;span style="color:hsl(270,75%,60%);"&gt;repudiare&lt;/span&gt; constituam &lt;s&gt;accommodare&lt;/s&gt;, vis ea sint &lt;u&gt;tempor&lt;/u&gt; eirmod. In mei eruditi moderatius, ex per iudico essent repudiandae. In vis vidisse menandri, an has repudiare disputationi delicatissimi. In has wisi justo postulant. Ei viderer ornatus mel, usu an partiendo tincidunt, ea eum brute ullamcorper.&lt;/p&gt;&lt;p&gt;10&lt;sup&gt;2&lt;/sup&gt;*5&lt;sup&gt;2&lt;/sup&gt; - H&lt;sub&gt;2&lt;/sub&gt;0&lt;/p&gt;&lt;p style="text-align:center;"&gt;&lt;strong&gt;Center: &lt;/strong&gt;Abhorreant reformidans eum an, per equidem repudiandae no, ex m elitr argumentum persequeris. In eam consul theophrastus delicatissimi, lobortis rationibus efficiendi mea ei, nam natum dolore scripserit ea. Sit partiendo eloquentiam ne. Sea no autem dolores, ridens gloriatur mel . Munere nullam epiri his id, dicam definiebas deterruisset eu vim. Ea veritus deleniti usu, nam ei idque choro volumus, ex nisl aliquam bonorum eam.&lt;/p&gt;&lt;p&gt;Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.&lt;/p&gt;&lt;p style="text-align:right;"&gt;&lt;strong&gt;Right:&lt;/strong&gt; Mel in lorem feugiat, tation dolore et eum. Id has utamur albucius molestiae. Dico justo maiestatis ei mei, labitur laboramus voluptatum nam ut. Saperet propriae nec an.&lt;/p&gt;&lt;p&gt;Quod oblique mei ne, cetero delicatissimi sea ,  est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim perseti vel ex. Mea nostrud instructior , et referrentur theophrastus eum. Pro in iuvaret senserit. In persius explicari conceptam nec, in populo ponderum consequat vis. Gloriatur conclusionemque no qui, vitae nominati nec te, laoreet torquatos accommodare ex qui. Eu natum mediocritatem eos, m ex alia oportere.&lt;/p&gt;&lt;p style="text-align:justify;"&gt;&lt;strong&gt;Justify:&lt;/strong&gt;  vis nonumes petentium disputationi, invidunt aliquando ex ius, orreret suscipiantur conclusionemque eu qui. Probo nemore ut est. Et eam dicta oratio eligendi. Quem dicta dolore vel et, at reque gloriatur definitionem sed.&lt;/p&gt;&lt;p&gt;Nec simul ullamcorper at. Mea possit dolores perseti te, ei quidam apeirian quo. Has quas corpora ne. Ius ad malis iusto delenit, sit ferri idque ex.&lt;/p&gt;&lt;blockquote&gt;&lt;p&gt;Blockquote: Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam perila neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dint mei ut. Pro no debet referrentur, ei est falli libris. Eu per summo salutandi hendrerit, salutatus democritum interesset nec ut. Wisi lobortis incorrupte ei mea, explicari interpretaris has et.&lt;/p&gt;&lt;/blockquote&gt;&lt;p&gt;A new line:&lt;/p&gt;&lt;hr&gt;&lt;p&gt;Now keep going…&lt;/p&gt;&lt;figure class="image"&gt;&lt;a href="https://github.com/Mathh19"&gt;&lt;img src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg" alt="O alt text da imagem"&gt;&lt;/a&gt;&lt;figcaption&gt;Aqui está uma imagem&lt;/figcaption&gt;&lt;/figure&gt;&lt;p&gt;Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu m, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id m agam clita nostro, tale tincidunt delicatissimi ei eos.&lt;/p&gt;&lt;figure class="image image-style-side"&gt;&lt;img src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg" alt="Uma imagem à direita"&gt;&lt;figcaption&gt;Uma imagem à direita&lt;/figcaption&gt;&lt;/figure&gt;&lt;p&gt;Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu m, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id m agam clita nostro, tale tincidunt delicatissimi ei eos.&lt;/p&gt;&lt;p&gt;Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu m, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id m agam clita nostro, tale tincidunt delicatissimi ei eos.&lt;/p&gt;&lt;p&gt;Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.&lt;/p&gt;&lt;p&gt; mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.&lt;/p&gt;&lt;p&gt;Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus m. Liber recteque duo no, ludus reprehendunt has an.&lt;/p&gt;&lt;p&gt;Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.&lt;/p&gt;&lt;p&gt; mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.&lt;/p&gt;&lt;p&gt;Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus m. Liber recteque duo no, ludus reprehendunt has an.&lt;/p&gt;&lt;p&gt;Bullet:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus m. Liber recteque duo no, ludus reprehendunt has an.&lt;/li&gt;&lt;li&gt;iriure posidonium consectetuer ad eam.&lt;/li&gt;&lt;li&gt;Nobis reprimique duo in, eam corpora mandamus&lt;/li&gt;&lt;li&gt;ex. In&lt;/li&gt;&lt;li&gt;quod euismod&lt;/li&gt;&lt;li&gt;eum, ne habemus cotidieque necessitatibus m.&lt;/li&gt;&lt;li&gt;Liber recteque duo no, ludus reprehendunt has an.&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;Now ordered:&lt;/p&gt;&lt;ol&gt;&lt;li&gt;Item 1&lt;/li&gt;&lt;li&gt;Item&lt;/li&gt;&lt;li&gt;Item&lt;/li&gt;&lt;li&gt;Item&lt;/li&gt;&lt;/ol&gt;&lt;p&gt;A think that's it…&lt;/p&gt;&lt;figure class="table"&gt;&lt;table&gt;&lt;thead&gt;&lt;tr&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;th&gt;Title 1&lt;/th&gt;&lt;/tr&gt;&lt;/thead&gt;&lt;tbody&gt;&lt;tr&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;td&gt;Item&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;&lt;/figure&gt;&lt;p&gt;A video:&lt;/p&gt;&lt;figure class="media"&gt;&lt;oembed url="https://www.youtube.com/watch?v=oLmRV25eQvo"&gt;&lt;/oembed&gt;&lt;/figure&gt;&lt;p&gt;The end.&lt;/p&gt;

          </code>
        </pre>
      </div>
    </div>
  </div>
</div>
`;
